feature/soptamp/src/main/java/org/sopt/official/stamp/designsystem/style/Theme.kt

```diff
@@ -283,10 +344,11 @@ fun ProvideSoptColorsAndTypography(colors: SoptColors, typography: SoptTypograph
 }
 
 @Composable
-fun SoptTheme(darkTheme: Boolean = false, content: @Composable () -> Unit) {
-    val colors = soptLightColors()
+fun SoptTheme( darkTheme: Boolean = isSystemInDarkTheme(), content: @Composable () -> Unit) {
```

Comment:
```suggestion
fun SoptTheme(darkTheme: Boolean = isSystemInDarkTheme(), content: @Composable () -> Unit) {
```

터미널에서 `./gradlew spotlessApply` 한번만 돌려주세요